web/wp-content/themes/dctx-v2/assets/bower_components/jquery/src/traversing/var/rneedsContext.js

```diff
@@ -0,0 +1,8 @@
+define( [
+	"../../core",
+	"../../selector"
+], function( jQuery ) {
```

Comment:
:no_entry_sign: **Error**: Only one argument is allowed per line in a multi-line function call (*PEAR.Functions.FunctionCallSignature.MultipleArguments*).